### ISSUE TITLE
customJenkinsLabels is not working for me in helm chart version 3.2.1

### ISSUE THREAD

**Issue by KamatR** [jenkinsci/helm-charts#371]
Helm Version:
version.BuildInfo{Version:"v3.5.4", GitCommit:"1b5edb69df3d3a08df77c9902dc17af864ff05d1", GitTreeState:"clean", GoVersion:"go1.15.11"}

Kubernetes Version:

Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.4", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"clean", BuildDate:"2021-02-18T16:12:00Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"clean", BuildDate:"2020-11-11T13:09:17Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}

**Which version of the chart**: 3.2.1

**What happened**:
I am adding the helm chart repo using below command with the chart version 3.2.1
helm repo add jenkinsci https://charts.jenkins.io

I have added the below configuration for master jenkins node.
customJenkinsLabels: dynamic
but this is not getting reflected in my jenkins web page.

**What you expected to happen**:
once I have added a below configuration in _helpers.tpl , the information started to get reflected in the jenkins web page.
labelString: "{{ .Values.controller.customJenkinsLabels }}"

why is customJenkinsLabels is not working in my case without my workaround ?? If the workaround is correct could you add this as a fix in next chart version ??

**How to reproduce it** (as minimally and precisely as possible):

values.yaml (only put values which differ from the defaults)
customJenkinsLabels: dynamic
numExecutors: 6
adminPassword: "password"
serviceType: NodePort
nodePort: 30176

key: value
customJenkinsLabels: dynamic

helm install my-release jenkins/jenkins --version version --values values.yaml
jusy try it by adding customJenkinsLabels to default values.yaml file and try to install it

**Anything else we need to know**:

**Comment by KamatR** [comment]
Hi,

anyone started looking into this issue ??

**Comment by hak33m16** [comment]
I just put a PR up for it, but I've got some questions in there for maintainers about how back releases/cherry picks for issues like this work because of the current release process. So as of now, the changes may only go into 3.3.x

**Comment by KamatR** [comment]
Hi,

As per the chart version is concerned, I was testing my end to end jenkins with chart version 3.2.1, but the issue is persistent on latest chart version too. I dont have specific requirement for chart version to be fixed.  I can test it on chart version where the fix will be available.

Please provide us a exact chart version to test once the review is approved.

Thanks and Regards,
Raghavi G. Kamat

**Comment by pashcan** [comment]
Still seeing this or related issue in chart v3.5.7

**Comment by hak33m16** [comment]
@pashcan Sorry about that, I took this on and then forgot about it. Just made the updates requested by the Jenkins team, hopefully it'll get into the next patch

**Comment by pashcan** [comment]
@hak33m16 No worries. I can still assign labels to main pod and agents through other variable so this is mostly nice-to-have for my use case. Thanks for looking into it! :bow:
